pom.xml

```diff
@@ -184,6 +184,12 @@ SOFTWARE.
       <artifactId>hamcrest</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>java-hamcrest</artifactId>
+      <version>2.0.0.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why this dependency is needed? Hamcrest website says that the only dependency is required: http://hamcrest.org/JavaHamcrest/distributables

Reply:
You are right. Already include in `org.hamcrest:hamcrest:2.2`